modules/ROOT/pages/deploy-maven-3.x.adoc

```diff
@@ -1,20 +1,27 @@
 = Deploy an Application to Runtime Fabric using Maven (Mule 3.x)
 
-Runtime Fabric supports using Maven to manage and deploy Mule applications using Maven. The information in this topic is specific to deploying an application to Mule runtime, version 3.x.
+Runtime Fabric supports using Maven to manage and deploy Mule applications using Maven. The information 
+in this topic is specific to deploying an application to Mule runtime, version 3.x.
 
 [NOTE]
-To deploy an application to Runtime Fabric, the application must be published in Exchange. You can use Studio, as describe in  xref:studio::export-to-exchange-task.adoc[Publishing a Project to Exchange]. Alternatively, use the Mule Maven Plugin. See xref:exchange::to-publish-assets-maven.adoc[Publish and Deploy Exchange Assets Using Maven] for more information.
-
+To deploy an application to Runtime Fabric, the application must be published in Exchange. 
+You can use Studio, as describe in  xref:studio::export-to-exchange-task.adoc[Publishing a Project to Exchange]. 
+Alternatively, use the Mule Maven Plugin. See 
+xref:exchange::to-publish-assets-maven.adoc[Publish and Deploy Exchange Assets Using Maven] for more information.
 
 == Prerequisites
 
-Before using Maven to deploy a Mule application, you should be familiar with Maven, managing pom.xml files, and working with Maven plugins.
+Before using Maven to deploy a Mule application, you should be familiar with Maven, managing pom.xml files, 
+and working with Maven plugins.
 
-Before deploying an application to Runtime Fabric you should understand the number of resource it requires. See xref:runtime-fabric::deploy-resource-allocation.adoc[Resource Allocation on Anypoint Runtime Fabric] for more information.
+Before deploying an application to Runtime Fabric, you should understand the number of resources it requires. 
```

Comment:
Is "it" from "number of resources it requires" referring to the application, or Runtime Fabric?

Reply:
Fixed.